content/faq/text-to-speech-android-tts/index.es.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Instrucciones de voz (Text-To-Speech, TTS) en Android
+description: Guía sobre cómo hacer que TTS funcione en Android
+
+taxonomies:
+  faq: ["Instrucciones de voz"]
+
+extra:
+  order: 10
+---
+
+## Información general
+
+Organic Maps utiliza el motor de texto a voz (Text-To-Speech, TTS) del sistema para las instrucciones de voz. Los motores predeterminados varían según el dispositivo, que puede ser el motor Text-To-Speech de Google, el motor del fabricante del dispositivo o un motor de terceros.
+
+La recomendación oficial de Organic Maps es [RHVoice](https://rhvoice.org/), que es un motor de voz gratuito y de código abierto que se puede descargar desde [Google Play](https://play.google.com/store/apps/details?id=com.github.olga_yakovleva.rhvoice.android) y [F-Droid](https://f-droid.org/en/packages/com.github.olga_yakovleva.rhvoice.android/).
+
+## Instrucciones
+
+- Abra la aplicación de Ajustes de su dispositivo Android.
+- Vaya a Ajustes Adicionales y después a Accesibilidad.
+- Elija su motor preferido, el tono y la velocidad de la voz.
+
+Si no puede encontrar estos ajustes, abra la aplicación de Ajustes y busque "Instrucciones de voz".
+
+Aviso: Note que estas instrucciones pueden variar en función de la marca de su dispositivo.
+
+Estas opciones pueden que no estén disponibles si no tiene la funcionalidad TTS (Instrucciones de voz) instaladas en su dispositivo. Por favor, revise la tabla más abajo para instalar cualquier motor que soporte su idioma.
+
+## Capturas de pantalla
+
+|             |             |             |
+| ----------- | ----------- | ----------- |
+| <img src="tts_config_1.jpg" width="400" alt="Ajustes TTS"> | <img src='tts_config_2.jpg' width='400' alt='Ajustes adicionales'> | <img src='tts_config_3.jpg' width='400' alt='Accesibilidad'> |
```

Comment:
1. Does it make sense to hard-code the width?
2. Does it make sense to use the markdown images syntax here and at the bottom?